documentation/book/ref-storage-persistent.adoc

```diff
@@ -71,6 +72,51 @@ storage:
 # ...
 ----
 
+== Storage class overrides
+
+{ProductName} allows to specify different storage class for each broker.
+This is useful for example when the storage classes are restricted to different availability zones / datacenters.
+You can use the `overrides` field for this purpose.
+
+.Example of {ProductName} cluster using the storage class overrides
+[source,yaml,subs="attributes+"]
+----
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  labels:
+    app: my-cluster
+  name: my-cluster
+  namespace: myproject
+spec:
+  # ...
+  kafka:
+    replicas: 3
+    storage:
+      deleteClaim: true
+      size: 100Gi
+      type: persistent-claim
+      class: my-storage-class
+      overrides:
+        - broker: 0
+          class: my-storage-class-zone-1a
+        - broker: 1
+          class: my-storage-class-zone-1b
+        - broker: 2
+          class: my-storage-class-zone-1c
+  # ...
+----
+
+In the example above, the default storage class is `my-storage-class`.
```

Comment:
This sentence could introduce (go above) the example. 'In this example, the default storage class is '`my-storage-class`.